datahub-web-react/src/app/ingest/source/builder/RecipeForm/constants.ts

```diff
@@ -0,0 +1,312 @@
+    rules: null,
+    section: 'Schemas',
+    getValueFromRecipe: (recipe: any) => recipe.source.config?.schema_pattern?.allow,
+    setValueOnRecipe: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, 'allow', 'schema_pattern'),
+};
+
+export const SCHEMA_DENY: RecipeField = {
+    name: 'schema_pattern.deny',
+    label: 'Deny Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    rules: null,
+    section: 'Schemas',
+    getValueFromRecipe: (recipe: any) => recipe.source.config?.schema_pattern?.deny,
+    setValueOnRecipe: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, 'deny', 'schema_pattern'),
+};
+
+export const VIEW_ALLOW: RecipeField = {
+    name: 'view_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    rules: null,
+    section: 'Views',
+    getValueFromRecipe: (recipe: any) => recipe.source.config?.view_pattern?.allow,
+    setValueOnRecipe: (recipe: any, values: string[]) => setListValuesOnRecipe(recipe, values, 'allow', 'view_pattern'),
+};
+
+export const VIEW_DENY: RecipeField = {
+    name: 'view_pattern.deny',
+    label: 'Deny Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    rules: null,
+    section: 'Views',
+    getValueFromRecipe: (recipe: any) => recipe.source.config?.view_pattern?.deny,
+    setValueOnRecipe: (recipe: any, values: string[]) => setListValuesOnRecipe(recipe, values, 'deny', 'view_pattern'),
+};
+
+export const TABLE_ALLOW: RecipeField = {
+    name: 'table_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    rules: null,
+    section: 'Tables',
+    getValueFromRecipe: (recipe: any) => recipe.source.config?.table_pattern?.allow,
+    setValueOnRecipe: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, 'allow', 'table_pattern'),
+};
+
+export const TABLE_DENY: RecipeField = {
+    name: 'table_pattern.deny',
+    label: 'Deny Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    rules: null,
+    section: 'Tables',
+    getValueFromRecipe: (recipe: any) => recipe.source.config?.table_pattern?.deny,
+    setValueOnRecipe: (recipe: any, values: string[]) => setListValuesOnRecipe(recipe, values, 'deny', 'table_pattern'),
+};
+
+export const RECIPE_FIELDS = {
+    [SNOWFLAKE]: {
+        fields: [ACCOUNT_ID, WAREHOUSE, USERNAME, PASSWORD, ROLE],
+        advancedFields: [
+            INCLUDE_LINEAGE,
+            IGNORE_START_TIME_LINEAGE,
+            CHECK_ROLE_GRANTS,
+            PROFILING_ENABLED,
+            STATEFUL_INGESTION_ENABLED,
+        ],
+        filterFields: [
+            TABLE_ALLOW,
+            TABLE_DENY,
+            DATABASE_ALLOW,
+            DATABASE_DENY,
+            SCHEMA_ALLOW,
+            SCHEMA_DENY,
+            VIEW_ALLOW,
+            VIEW_DENY,
+        ],
+    },
+};
+
```

Comment:
so cool how simple this is

Reply:
very easy to add / remove fields or whole new connectors now!